index.html

```diff
@@ -44,9 +44,16 @@
       </div>
 
       <div class="carousel">
-        <ul class="carousel__list testimonials-carousel">
+        <input type="radio" name="testimonials" id="item-1" checked>
```

Comment:
Please, use CSS scroll snap instead of input type radio.

`cursor: grab`

Align center
![image](https://github.com/katatsu12/live_blog/assets/22080708/ca534d79-38bf-4f3c-bd90-8bd0997ba7c9)